### PR TITLE
Add a page about our open source stuffs

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Register page
- [x] Content
- [x] Design
- [x] Add to navs

### CODE REVIEW

**Comment by sn3p** [comment]
Think we can just make it an English only page? (at least the content)

**Comment by RonaldDijkstra** [comment]
This page should be more than just a list

**Comment by RonaldDijkstra** [comment]
Btw thats not a comment on your remark @sn3p

**Comment by sn3p** [comment]
Inspiration maybe? https://bocoup.com/open-source

**Comment by RonaldDijkstra** [comment]
Awesome!

**Comment by RonaldDijkstra** [comment]
open-source

**Comment by RonaldDijkstra** [comment]
@sn3p would you mind checking the content please? :-)

To the others, what do you think?

**Comment by GerwinV** [comment]
Can we use the Phoenix logo for the `ex_cell` module? It is an elixir module, but for use in Phoenix. Nice to have the Phoenix logo there as well.

Looking good!

**Comment by sn3p** [comment]
Could get custom logos for our repos 😆 
http://openlogos.org

**Comment by RonaldDijkstra** [comment]
We sure can @gerwinv

**Comment by sn3p** [comment]
PickupGameBot has a logo:

https://github.com/DefactoSoftware/PickupGameBot/blob/master/app/assets/images/bot.svg

**Comment by sn3p** [comment]
> Can we use the Phoenix logo for the `ex_cell` module? It is an elixir module, but for use in Phoenix. Nice to have the Phoenix logo there as well.

Maybe also add a `Phoenix` label?

File: source/localizable/open-source.html.erb [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+description: Defacto relies on open source software and likes to give back to the community.
+---
+
+
+<div class="hero">
+
+  <div class="hero-inner">
+
+    <h1>
+      Developing <em>Software</em>!
```
Comment (sn3p) [inline source/localizable/open-source.html.erb]: Personally I don't like italic here 🙈

File: source/localizable/open-source.html.erb [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    </h1>
+
+    <article>
+
+      <p>
+        Defacto relies on open source software and likes to give back to the community.
+      </p>
+
+      <p>
+         Among other things, we are sponsor of <a href="https://elixirforum.com/" target="_blank" rel="noopener" title="Elixir Forum">the Elixir forum</a> and contribute to the community through our open source libraries.
```
Comment (sn3p) [inline source/localizable/open-source.html.erb]: Something like this maybe? (the 2 paragraphs combined)

> Defacto maintains and contributes to open source software and libraries. We rely on open source software and like to give back to the community. Among other initiatives, we are proud sponsor of <a href="https://elixirforum.com/" target="_blank" rel="noopener" title="Elixir Forum">Elixir Forum</a>.

```html
Defacto maintains and contributes to open source software and libraries. We rely on open source software and like to give back to the community. Among other initiatives, we are proud sponsor of <a href="https://elixirforum.com/" target="_blank" rel="noopener" title="Elixir Forum">Elixir Forum</a>.
```

**Comment by RonaldDijkstra** [comment]
All good. Anything else? 😆

**Comment by fatboypunk** [comment]
> All good. Anything else? 😆

Update the screenshot plz 💃

**Comment by RonaldDijkstra** [comment]
screeeeenshit

**Comment by fatboypunk** [comment]
The text on top doesn't have enough contrast with the background (#accessibility) 🙊

File: source/localizable/open-source.html.erb [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+        <div class="work-desc">
+
+          <h3 class="work-header">
+            <a href="https://github.com/DefactoSoftware/test_selector" target="_blank" title="Learn more about test_selector" rel="noopener">
+              test_selector
+            </a>
+          </h3>
+
+          <p>
+            Elixir library to help selecting the right elements in your tests.
```
Comment (sn3p) [inline source/localizable/open-source.html.erb]: maybe?

> Library with Elixir and Hound helpers to set/get test selectors to/from elements in a Phoenix app.

**Comment by RonaldDijkstra** [comment]
higher-contrast

File: source/localizable/open-source.html.erb [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+          </p>
+
+          <p class="work-meta">
+            <span class="work-language-color elixir"></span>
+            <span>Elixir</span>
+          </p>
+
+          <a class="button dark small" href="https://github.com/DefactoSoftware/test_selector" target="_blank" title="Learn more about test_selector" rel="noopener">Learn more</a>
```
Comment (sn3p) [inline source/localizable/open-source.html.erb]: `Visit` as label?

File: source/localizable/open-source.html.erb [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+        <div class="work-desc">
+
+          <h3 class="work-header">
+            <a href="https://github.com/DefactoSoftware/cell-js" target="_blank" title="Learn more about cell-js" rel="noopener">
+              cell-js
+            </a>
+          </h3>
+
+          <p>
+            Bind JavaScript to specific templates or serverside components.
```
Comment (sn3p) [inline source/localizable/open-source.html.erb]: > Simple and lightweight solution for binding Javascript to specific templates or side server components.

(from the repo readme)

**Comment by RonaldDijkstra** [comment]
https://staging.defacto.nl/open-source/

File: source/localizable/open-source.html.erb [hunk 6]
```diff
@@ -0,0 +1,286 @@
...
+        <div class="work-desc">
+
+          <h3 class="work-header">
+            <a href="https://github.com/DefactoSoftware/PickupGameBot" target="_blank" title="Learn more about PickupGamebot" rel="noopener">
+              PickupGameBot
+            </a>
+          </h3>
+
+          <p>
+            To help us set up pickup futsal games and maybe more
```
Comment (sn3p) [inline source/localizable/open-source.html.erb]: Add dot 👀

File: source/localizable/open-source.html.erb [hunk 7]
```diff
@@ -0,0 +1,286 @@
...
+
+  </div>
+
+</section>
+
+<section class="visit-us gray">
+
+  <div class="row">
+
+    <a class="button big" href="https://github.com/DefactoSoftware/" target="_blank" rel="noopener" title="Vist us on GitHub">Visit us on GitHub <span class="icon-github"></span></a>
```
Comment (sn3p) [inline source/localizable/open-source.html.erb]: > For more Open Source Projects [visit us on GitHub](https://github.com/DefactoSoftware/)

File: source/stylesheets/base/_variables.scss [hunk 8]
```diff
@@ -38,6 +38,10 @@ $gray-menu: #444;
 $pink: #ec677a;
 $capp-color: #393e46;
 $capp-dark-blue: #353741;
+$elixir: #6e4a7e;
```
Comment (sn3p) [inline source/stylesheets/base/_variables.scss]: Maybe prefix these with `$tag-`?

File: source/localizable/open-source.html.erb (hunk 3 above)
Comment (RonaldDijkstra) [reply source/localizable/open-source.html.erb]: I just copied all these from GitHub, maybe update them as well?

**Comment by sn3p** [comment]
Few other suggestions:

1. add hover to title link
2. add `Elixir` tag to ex_cell
3. add `Phoenix` tag to ex_css_modules
4. black button to dark/heavy?

File: source/localizable/open-source.html.erb (hunk 3 above)
Comment (sn3p) [reply source/localizable/open-source.html.erb]: The text I put here is from the readme, but maybe the one you used is better... up to you!

**Comment by RonaldDijkstra** [comment]
All suggestions 💚 

Updated https://staging.defacto.nl/open-source/

**Comment by sn3p** [comment]
Elixir tag has wrong color for ex_cell:

screen shot 2018-10-09 at 17 31 49

File: source/localizable/open-source.html.erb [hunk 9]
```diff
@@ -0,0 +1,290 @@
...
+  </div>
+
+</div>
+
+<section class="work">
+
+  <div class="row">
+
+    <h2>
+      A selection of our open source repositories:
```
Comment (sn3p) [inline source/localizable/open-source.html.erb]: projects instead of repositories?

> A selection of our open source projects:

Comment (RonaldDijkstra) [reply source/localizable/open-source.html.erb]: Döner

**Comment by RonaldDijkstra** [comment]
Yes

**Pull request merged**